install-config.yaml

```diff
@@ -19,13 +19,13 @@ metadata:
   name: crc
 networking:
   clusterNetwork:
-  - cidr: 10.128.0.0/14
+  - cidr: 10.116.0.0/14
     hostPrefix: 23
   machineNetwork:
   - cidr: 192.168.126.0/24
   networkType: OpenShiftSDN
   serviceNetwork:
-  - 172.30.0.0/16
+  - 172.25.0.0/16
 platform:
```

Comment:
I assume these values were just picked "randomly"?

Reply:
yes these are randomly picked.

Reply:
we could choose a single IP block (large enough) and say explicitly in the documentation that this IP block is reserved for crc. 
Inside this block, we can do what we want.